test/Microsoft.Extensions.CodeGeneration.Sources.Test/LibraryExportExtensionsTest.cs

```diff
@@ -29,13 +27,12 @@ public void LibraryExportExtensions_GetMetadataReferences()
             var refList = _export.GetMetadataReferences();
             Assert.True(refList.Any());
         }
-        
+
         [Fact]
         public void LibraryExportExtensions_GetMetadataReferences_throwException()
         {
-            //arrange
             var libraryExporter = GetInvalidLibraryExporter();
-            _export = libraryExporter.GetAllExports().Last();
```

Comment:
Maybe this used to work when the order of the library exports were reversed. Currently, it will get `System.Xml.XDocument` or some other library which may be valid. Since it can get an arbitrary export, the test can break when it expect the library to be non-existent.